src/deployment.pri

```diff
@@ -56,7 +56,7 @@ isEmpty(QMLPLUGINDUMP): QMLPLUGINDUMP = 1
 equals(QMLPLUGINDUMP, 1): !ios: !android: release: {
     dumppluginqmltypes.CONFIG = no_files no_path
     dumppluginqmltypes.commands = $$dirname(QMAKE_QMAKE)/qmlplugindump -nonrelocatable "$$uri $$PLUGIN_VERSION $$shell_path($$OUT_PWD/../../imports/) > $$shell_path($$OUT_PWD/../../imports/$$TARGETPATH/plugins.qmltypes)"
-    INSTALLS += dumppluginqmltypes
+    # INSTALLS += dumppluginqmltypes
```

Comment:
So that I can use ``qmake -r `"QMLPLUGINDUMP=0"``to disable QMLPLUGINDUMP , right?